tests/Commands/HealthCheckMakeCommandTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Tests\Commands;
+
+use Tests\TestCase;
+use Illuminate\Support\Facades\File;
+
+class HealthCheckMakeCommandTest extends TestCase
+{
+    public function getPackageProviders($app)
+    {
+        return ['UKFast\HealthCheck\HealthCheckServiceProvider'];
+    }
+
+    /**
+     * @test
+     */
+    public function creates_a_new_check()
+    {
+        $checkName = "TestCheck";
+        $checkClassFile = $this->app->basePath("app/Checks/{$checkName}.php");
```

Comment:
This is the other thing I'm not sure about, would we be able to put this in a tests/stubs folder which we commit to `.gitignore`

If the test fails before it gets a chance to unlink it, we could end up with these test stubs accidentally comitted

Reply:
In fact, ignore this. Not like there are any checks in the app folder of this project

Reply:
What we can do is to check for the file and unlink it before creating and asserting it.